pkg/webhooks/generate/generate.go

```diff
@@ -145,7 +145,7 @@ func retryApplyResource(client *kyvernoclient.Clientset, grSpec kyverno.Generate
 				"policyName":        grSpec.Policy,
 				"resourceName":      grSpec.Resource.Name,
 				"resourceKind":      grSpec.Resource.Kind,
-				"ResourceNamespace": grSpec.Resource.Namespace,
+				"resourceNamespace": grSpec.Resource.Namespace,
 			}))
 			grList, err := grLister.List(selector)
 			if err != nil {
```

Comment:
Cannot add a comment to line 157, put it here.
As we are fetching grList by the above labels, do we still need this comparison?
```
if grSpec.Policy == v.Spec.Policy && grSpec.Resource.Name == v.Spec.Resource.Name && grSpec.Resource.Kind == v.Spec.Resource.Kind && grSpec.Resource.Namespace == v.Spec.Resource.Namespace {"
```

Reply:
No the comparison is not necessary. Will make appropriate changes.